appinventor/docs/reference/components/userinterface.html

```diff
@@ -1377,7 +1377,10 @@ <h3>Properties</h3>
 </dl>
 
 <h3>Events</h3>
-none
+<dl>
+  <dt><code>WebViewStringChange()</code></dt>
```

Comment:
I went and double-checked this. We do have a text parameter `value` that is used to report the new value. That should be documented in the parameter list.

Reply:
@ewpatton oops overlooked that. Could you check the new commit?